Misc/NEWS.d/next/Documentation/2024-11-27-22-56-48.gh-issue-127347.xyddWS.rst

```diff
@@ -0,0 +1 @@
+Document :func:`traceback.print_list`.
```

Comment:
Could you please write something similar to what we did in https://github.com/python/cpython/pull/127304?
Otherwise LGTM.

Reply:
Will do :) There's also been some discussion on the issue if you want to read it: https://github.com/python/cpython/issues/127347#issuecomment-2507416071 Serhiy suggested this _might_ need a `versionadded` directive or a What's New entry. What do you think?